d2networking/d2client/d2localclient/local_client_connection.go

```diff
@@ -57,7 +59,10 @@ func (l *LocalClientConnection) Open(connectionString string, saveFilePath strin
 
 // Close disconnects from the server and destroys it.
 func (l *LocalClientConnection) Close() error {
-	l.SendPacketToServer(d2netpacket.CreateServerClosedPacket())
+	err := l.SendPacketToServer(d2netpacket.CreateServerClosedPacket())
+	if err != nil {
+		log.Printf("LocalClientConnection: error sending ServerClosedPacket to server: %s", err)
```

Comment:
What's the point of swallowing the error? I'd prefer to see the lint warnings over swallowing it, unless we truly do just want to ignore it.

Reply:
I was under the impression we were trying to remove all lint warnings. I will be doing more work on d2networking during which I will have a proper look at the error handling.

Reply:
On second thoughts, I could have just handled the errors as opposed to using the blank identifier. I will fix that.